BLaurenB/techtime
# Category Show Mimics Freelancer Index with Cards for Freelancers

#### What's this PR do?
applies the card structure to the category show in the same way as the freelancers index.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
category show

#### How should this be manually tested?
start server, click on categories, click on a specific category. the freelancers in this category should be in the cards.

#### Any background context you want to provide?
#### What are the relevant tickets?
Closes #86 
#### Screenshots (if appropriate)
#### Questions:
The images for a specific freelancer are only showing up on the freelancer index; they aren't showing up in any other pages

I added a card to capture this image issue.
